datamover/pkg/drivers/lifecycle/incloud_transition.go

```diff
@@ -119,7 +119,7 @@ func getStorageClassName(tier int32, storageType string) (string, error) {
 	case flowtype.STOR_TYPE_CEPH_S3:
 		key = backend.BackendTypeCeph
 	case flowtype.STOR_TYPE_GCP_S3:
```

Comment:
any specific reason we need this 3 way mapping of
int <to> const <to> backend string ? can we just use the const-key everywhere ?

Reply:
this problem has gone, please check it again, thanks.